full-stack-tests/core/src/frontend/hub/ExternalTextures.test.ts

```diff
@@ -100,9 +107,34 @@ describe("external texture requests (#integration)", () => {
     });
   }
 
+  async function testDownsamplingTextures() {
+    const maxTextureSize = 8;
+    for (const name of goodTexNames) {
+      // check that requested textures are downsampled to maxTexturesize when requested.
+      let texBytes = await imodel.getTextureImage({ name, maxTextureSize });
+      expect(texBytes).to.not.be.undefined;
+      let imageSource = new ImageSource(texBytes!, ImageSourceFormat.Jpeg);
+      let image = await imageElementFromImageSource(imageSource);
+      expect(image.width).to.be.lessThanOrEqual(maxTextureSize);

```

Comment:
Should probably verify that at least one of width or height is exactly equal to maxTextureSize.

Reply:
@pmconne Good addition.  Added.